src/common/admin_socket.h

```diff
@@ -36,15 +36,56 @@ inline constexpr auto CEPH_ADMIN_SOCK_VERSION = "2"sv;
 
 class AdminSocketHook {
 public:
-  // NOTE: the sync handler doesn't take an input buffer currently because
-  // no users need it yet.
+  /**
+   * @brief
+   * Handler for admin socket commands, synchronous version
+   *
+   * Executes action associated with admin command and returns byte-stream output @ref out.
+   * There is no restriction on output. Each handler defines output semantics.
+   * Typically output is textual representation of some ceph's internals.
+   * Handler should use provided formatter @ref f if structuralized output is being produced.
+   *
+   * @param command[in] String matching constant part of cmddesc in @ref AdminSocket::register_command
```

Comment:
s/ref/c/

see http://www.doxygen.nl/manual/commands.html#cmdc and http://www.doxygen.nl/manual/commands.html#cmdref .

Reply:
@tchaikov Done.I still left \ref in places where it referred methods. You like it?